rackcity/api/serializers/asset_serializers.py

```diff
@@ -157,7 +157,7 @@ class RecursiveAssetSerializer(serializers.ModelSerializer):
     chassis_slot = RCIntegerField(
         allow_null=True, max_value=2147483647, min_value=0, required=False
     )
-    offline_storage_site = SiteSerializer()
+    offline_storage_site = serializers.SerializerMethodField()
```

Comment:
claire has also been makin asset serializers changes so watch out for merge conflicts